internal/pkg/store/project.go

```diff
@@ -98,3 +102,12 @@ func (s *Store) ListProjects() ([]*archer.Project, error) {
 	}
 	return projects, nil
 }
+
+func hostedZoneExist(hostedZones []*route53.HostedZone, domain string) bool {
+	for _, hostedZone := range hostedZones {
+		if domain == aws.StringValue(hostedZone.Name) || domain+"." == aws.StringValue(hostedZone.Name) {
```

Comment:
nit: You can also use [strings.HasPrefix](https://golang.org/pkg/strings/#HasPrefix) for roughly the same functionality